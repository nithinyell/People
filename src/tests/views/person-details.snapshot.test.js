import React from 'react';
import renderer from 'react-test-renderer';
import PersonDetails from '../../modules/person-details/view/person-details';

// todo - fix 
// seeing /People/node_modules/@react-navigation/elements/lib/commonjs/assets/back-icon.png:1
// ({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,jest){ï¿½PNG
// SyntaxError: Invalid or unexpected token

// test('renders correctly', () => {
//   const tree = renderer.create(<PersonDetails />).toJSON();
//   expect(tree).toMatchSnapshot();
// });