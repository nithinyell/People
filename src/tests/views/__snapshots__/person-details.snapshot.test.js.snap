// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PersonDetails renders correctly 1`] = `
<div>
  Error: Something went wrong.
</div>
`;
